README.md

```diff
@@ -1,49 +1,72 @@
+  </a>
+</p>
+
+---
 
-[libSQL](https://turso.tech/libsql) is an open source, open contribution fork of SQLite, created and maintained by [Turso](https://turso.tech). We aim to evolve it to suit many more use cases than SQLite was originally designed for, and plan to use third-party OSS code wherever it makes sense.
```

Comment:
I would like to mention are are `open contribution fork of SQLite` since sqlite isn't

Reply:
@notrab I just noticed you have already mentioned this in the header. So you can ignore my comment